kf_lib_data_ingest/validation/validation.py

```diff
@@ -26,53 +47,52 @@ def __init__(self, output_dir=None, setup_logger=True):
 
         if setup_logger:
```

Comment:
"setup" is a noun. "set up" is the verb form. See https://github.com/kids-first/kf-lib-data-ingest/pull/493/commits/d5f8a4fa416f2d3957cb3f70a5832c1d3aa07363

Reply:
Lol okie